vultr/terraform/variables.tf

```
@@ -0,0 +1,58 @@
+variable "api_key" {
+  description = "This is the Vultr API key"
+  type        = string
+}
+
+variable "region" {
+  description = "The region your VKE cluster will be deployed in"
+  type        = string
+  default     = "ewr"
+}
+
+variable "label" {
+  description = "The VKE cluster's label."
+  type        = string
+  default     = "qdrant-example"
+}
+
+variable "k8s_version" {
+  description = "The version of your VKE cluster you want deployed"
+  type        = string
+  default     = "v1.29.1+1"
```

This is already outdated

```suggestion
  default     = "v1.29.2+1"
```